src/gamestates/cCreditsState.cpp

```diff
@@ -307,6 +319,12 @@ void cCreditsState::prepareCrawlerLines()
         .color = Color::white(),
         .height = fontHeightWithALittlePadding
     });
+    m_lines.push_back(s_CreditLine {
+        .name = "Mira",
+        .txt = "for the pride of working on a big project in C++",
```

Comment:
Perhaps we should add a separate section for special messages by contributors; now your name would appear twice under "special thanks". I think this deserves a special place; you can leave a personal note to the audience then? wdyt?

Reply:
Delete my last comment; it should not appear in contributions.
Contributing means contributing to a project. This is not some kind of thank you.